lib/services/services_test.go

```diff
@@ -108,7 +108,7 @@ func (s *ServicesSuite) TestLabelKeyValidation(c *check.C) {
 func TestServerDeepCopy(t *testing.T) {
 	t.Parallel()
 	// setup
-	now := time.Date(1984, time.April, 4, 0, 0, 0, 0, time.UTC)
+	now := time.Date(1984, time.April, 4, 0, 0, 0, 0, time.Local)
```

Comment:
This tests the proto-based implementation deficiency (if we continued to use the implementation).

Reply:
Reverted to use UTC again as this triggers panic.